store/badger.go

```diff
@@ -20,10 +21,23 @@ type badgerTransaction struct {
 	tx *badger.Txn
 }
 
+func sanitizePassphrase(origin []byte) []byte {
```

Comment:
Rather than sanitizing the provided key, i'd prefer we just hash it with a 256-bit SHA hash so that whatever the user provides, it's gonna end up 256-bits (32 bytes) long (even for shorter passphrases).